src/Ionide.ProjInfo.Tool/Program.fs

```diff
@@ -80,15 +80,18 @@ let main argv =
             exit 1
         | projects ->
             if args.Contains Fcs then
-                let projects = projects |> List.map (fun p -> Ionide.ProjInfo.FCS.mapToFSharpProjectOptions p projects)
+                let projects = projects |> List.map (fun (p, po) -> Ionide.ProjInfo.FCS.mapToFSharpProjectOptions p (Seq.map fst projects))
 
                 if shouldSerialize then
                     projects |> Newtonsoft.Json.JsonConvert.SerializeObject |> printfn "%s"
                 else
                     printfn "%A" projects
 
             else if shouldSerialize then
-                projects |> Newtonsoft.Json.JsonConvert.SerializeObject |> printfn "%s"
+                projects |> Seq.map fst |> Newtonsoft.Json.JsonConvert.SerializeObject |> printfn "%s"
+                printfn "----"
+                projects |> Seq.map snd |> Seq.collect (fun x -> x.Items) |> Seq.map(fun ii -> ii.ItemType, ii.EvaluatedInclude) |> Newtonsoft.Json.JsonConvert.SerializeObject |> printfn "%s"
+                projects |> Seq.map snd |> Seq.collect (fun x -> x.Properties) |> Seq.map(fun ii -> ii.Name, ii.EvaluatedValue)  |> Newtonsoft.Json.JsonConvert.SerializeObject |> printfn "%s"
```

Comment:
Added ability to get the `ProjectInstance` `Items` and `Properties` for further debugging.

Reply:
Maybe should be behind another flag?